LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Denis Bourgeois & Dan Macumber
+Copyright (c) 2020-2022 Denis Bourgeois & Dan Macumber
```

Comment:
A bit more clarity on the range of copyright period.
Copied/pasted license on TBD lib/gem files.